stellar-multitenant/internal/httphandler/tenants_handler.go

```diff
@@ -97,9 +98,28 @@ func (h TenantsHandler) Post(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	var tntBaseURL string
+	if reqBody.BaseURL != nil {
+		tntBaseURL = *reqBody.BaseURL
+	} else {
+		tntBaseURL, err = utils.GenerateTenantURL(h.BaseURL, tnt.Name)
+		if err != nil {
+			httperror.InternalError(ctx, "Could not generate tenant URL", err, nil).Render(rw)
+			return
+		}
+	}
+
+	var tntSDPUIBaseURL string
+	if reqBody.SDPUIBaseURL != nil {
+		tntSDPUIBaseURL = *reqBody.SDPUIBaseURL
+	} else {
+		tntSDPUIBaseURL = tntBaseURL
```

Comment:
SDPUIBaseURL should generated the same way as BaseURL above.
tntSDPUIBaseURL, err = utils.GenerateTenantURL(h.SDPBUIBaseURL, tnt.Name)

Reply:
thanks for the catch! I addressed it here [708a4a3](https://github.com/stellar/stellar-disbursement-platform-backend/pull/281/commits/708a4a36a972b2ec2c325c6901a224a5dc01bf0f)

Reply:
added here! [d2b153d](https://github.com/stellar/stellar-disbursement-platform-backend/pull/281/commits/d2b153d7d3fda0f1dd53857e454491ccc0c201d4)